src/Core/Containers/VectorArray.hpp

```diff
@@ -26,68 +41,42 @@ class VectorArray : public AlignedStdVector<V>
     /// Returns the array as an Eigen Matrix Map
     MatrixMap getMap() {
         CORE_ASSERT( !this->empty(), "Cannot map an empty vector " );
-        return MatrixMap( this->data()->data(), V::RowsAtCompileTime, this->size() );
+        return MatrixMap( TypeHelper::getData( this ),
```

Comment:
Use `static_assert`to ensure that `TypeHelper::NumberOfComponents>1`

Reply:
done with enabe_if